reference/routes-yaml.md

```diff
@@ -178,6 +177,19 @@ You will see a route such as `https://*---add-theme-vmwklxcpbi6zq.eu.platform.sh
 
 [You can find detailed information about caching here](cache.html).
 
+##WebSocket routes
+
+To use WebSocket on a route, `cache` must be disabled. WebSocket is incompatible with buffering which is required for cache.
+Here is an example to define a route for serving WebSocket:
+
+```yaml
+"http://{default}/ws":
+    type: upstream
+    upstream: "websocket:http"
```

Comment:
@Crell "upstream" is part of our YAML schema. When you have "type: upstream", we look for the "upstream" key to know where the route should go.

As for the values, "websocket" is the name of the application, while "http" is the name of the endpoint.